src/AddressRegistry.Api.BackOffice/Validators/ValidationErrorCodes.cs

```diff
@@ -18,6 +18,7 @@ public static class ValidationErrorCodes
         public const string AddressCannotBeApproved = "AdresGehistoreerdOfAfgekeurd";
         public const string AddressCannotBeRejected = "AdresGehistoreerdOfGoedgekeurd";
         public const string AddressCannotBeDeregulated = "AdresGehistoreerdOfAfgekeurd";
+        public const string AddressCannotBeRegularized = "AdresGehistoreerdOfAfgekeurd";
```

Comment:
is this correct? (same as AddressCannotBeDeregulated)

Reply:
Yes, see ticket 3362
```
Een POST op {{adressen/<id>/acties/regulariseren}} voor een adres met status 
afgekeurd OF gehistoreerd 
Geeft als resultaat 400
```

The error code is in line with the other error codes